packages/okta-vue/package.json

```diff
@@ -8,15 +8,33 @@
     "src"
   ],
   "scripts": {
+    "prebuild": "npm run build:package-info",
     "prestart": "npm run build",
     "pretest": "npm run build && npm run build:harness",
     "prepublish": "npm run build",
-    "test": "npm run --prefix test/e2e/harness/ test",
+    "jest": "jest src/",
+    "test": "npm run jest && npm run --prefix test/e2e/harness/ test",
     "start": "npm run --prefix test/e2e/harness/ start",
     "build": "rimraf dist/ && cross-env NODE_ENV=production webpack --config webpack.config.js --output-library-target=umd -p",
     "build:harness": "npm --prefix test/e2e/harness install",
+    "build:package-info": "node ../../util/write-package-info.js . src/packageInfo.js",
```

Comment:
We should consider using [`lerna run`](https://github.com/lerna/lerna#run) to handle this for us.

Reply:
This task needs to be integrated into the build pipeline of each package, and ran locally while working on that specific package, so I'm not sure how lerna would help here?